src/game/client/neo/ui/neo_root.cpp

```diff
@@ -480,11 +480,13 @@ void CNeoRoot::FireGameEvent(IGameEvent *event)
 	}
 }
 
+#include "mp3player.h";
```

Comment:
As per the Valve memory debug shenanigans at https://github.com/NeotokyoRebuild/neo/blob/17a3f0c89ba53a171fa99248046185f4488fbadf/src/game/client/neo/ui/neo_root.cpp#L31

I think we should do this include above that line, ideally.